runtime/queries/column_numeric_histogram.go

```diff
@@ -149,6 +149,11 @@ func (q *ColumnNumericHistogram) calculateFDMethod(ctx context.Context, rt *runt
 		return fmt.Errorf("not available for dialect %q", olap.Dialect())
 	}
 
+	if olap.Dialect() == drivers.DialectClickHouse {
+		// Returning early with empty results because this query tends to hang on ClickHouse.
```

Comment:
just confirming - this is intentional ?

Reply:
Yeah, should probably have been a separate PR, but we want to keep that.